pulp_2to3_migration/app/plugin/docker/repository.py

```diff
@@ -57,3 +57,41 @@ def migrate_to_pulp3(cls, pulp2distributor, repo_version):
         base_config['repository_version'] = repo_version
         distribution, created = ContainerDistribution.objects.update_or_create(**base_config)
```

Comment:
this needs to be changed to:
`update_or_create(name=name, base_path=base_path, defaults=base_config)`